python/lsst/ts/genericcamera/driver/zwocamera.py

```diff
@@ -69,6 +71,39 @@ def initialise(self, config):
             # self.zwoDev.setPosition(self.filterNumber)
             self.filterNumber = self.zwoLib.getPosition(self.filterId)
 
+    def get_config_schema(self):
+        return yaml.safe_load(
+            """
+$schema: http://json-schema.org/draft-07/schema#
+description: Schema for the ZWO GenericCamera.
+type: object
+properties:
+  id:
+    default: 0
+    type: number
+  binValue:
+    default: 1
+    type: number
+  currentImageType:
+    default: Raw16
+    type: string
+    enum:
+      - Raw8
+      - RGB24
+      - Raw16
+      - Y8
+  useZWOFilterWheel:
```

Comment:
I make it snake_case and kept the `zwo` part since there are filter wheels of other vendors and this camera doesn't work with them. This way is is clear that a ZWO filter wheel is expected.